ansible/roles/nginx/vars/riakcs.yml

```diff
@@ -23,6 +23,7 @@ nginx_sites:
        proxy_read_timeout: 90
        proxy_buffer_size: 64k
        proxy_buffers: 8 64k
+       proxy_buffer_size: 64k
```

Comment:
This is double defined

Reply:
Oh dear, I was distracted and put this in the wrong file. Thanks